.github/workflows/compilation.yml

```diff
@@ -200,9 +205,6 @@ jobs:
         automatic_release_tag: "latest"
         title: "${{ steps.version.outputs.version }}"
```

Comment:
As I understand this naming should be extended for Mega uploads. I suggest adding additional step where ps2dev/ps2sdk git commit is checked and adding it into the title. Or if it is too hard, you can just use action id.

Reply:
> As I understand this naming should be extended for Mega uploads. I suggest adding additional step where ps2dev/ps2sdk git commit is checked and adding it into the title. Or if it is too hard, you can just use action id.

in order to pass the commit short hash we must modify the workflows of all repos that can trigger ps2dev actions forwader.

Forwarder included too...

Then, the workflows with workflow dispatch support must be updated to use the hash from the payload

Reply:
Why? Just do git clone as an additional step in opl workflow, thats all

Reply:
> Why? Just do git clone as an additional step in opl workflow, thats all

Reading from workflow dispatch payload instead of cloning all repos seems to be more reasonable to me...